docs/HowTo/Get-Started/Start-Web3Signer.md

```diff
@@ -8,16 +8,16 @@ description: Getting started with Web3Signer
 
 * [Signing key configuration files] to access the required signing keys.
 
-Web3Signer supports the Ethereum 1, Ethereum 2, and Filecoin platforms, so you must specify the
+Web3Signer supports the execution layer (formerly Ethereum 1.0), consensus layer (Ethereum 2.0), and Filecoin platforms, so you must specify the
 signing mode, and the location of the signing key configuration files when starting Web3Signer.
 
-=== "Ethereum 1.0"
+=== "Execution layer"
 
     ```bash
     web3signer --key-store-path=/Users/me/keyFiles/ eth1
```

Comment:
should we update the CLI options here too to reflect the wording changes?

Reply:
I added a clarification about eth1 and eth2 on the subcommand page. I don't think we need to change the CLI options. The Teku team is leaving them as is and just used the same type of clarification.